### PR TITLE
NOJIRA - Add authorship to existing blog posts

### PR DESCRIPTION
**Pull request by ChrisWiegman**
## Tasks

- [ ] I have signed a [Contributor License Agreement (CLA)](https://github.com/wpengine/faustjs#contributor-license-agreement) with WP Engine.

### CODE REVIEW

File: internal/faustjs.org/blog/2022-12-22-sprint-25-update.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 title: Sprint 25 Update
 description: Updates from the team's work on Sprint 25
 slug: sprint-25-update
+authors:
+  - name: Chris Wiegman
+    title: Engineering Manager
+    url: https://github.com/ChrisWiegman
+    image_url: https://avatars.githubusercontent.com/u/394675?v=4
```
Comment (TeresaGobble) [inline internal/faustjs.org/blog/2022-12-22-sprint-25-update.md:9]:
```suggestion
    author_image_url: https://avatars.githubusercontent.com/u/394675?v=4
```

File: internal/faustjs.org/blog/2023-01-04-faust-update-jan-04-2023.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 title: Faust Update - January 4, 2023
 description: Updates on Faust development for Jan 4, 2023
 slug: faust-update-jan-04-2023
+authors:
+  - name: Chris Wiegman
+    title: Engineering Manager
+    url: https://github.com/ChrisWiegman
+    image_url: https://avatars.githubusercontent.com/u/394675?v=4
```
Comment (TeresaGobble) [inline internal/faustjs.org/blog/2023-01-04-faust-update-jan-04-2023.md:9]:
```suggestion
    author_image_url: https://avatars.githubusercontent.com/u/394675?v=4
```

File: internal/faustjs.org/blog/2023-01-18-faust-update-jan-18-2023.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
 title: Faust Update - January 18, 2023
 description: Updates on Faust development for Jan 18, 2023
 slug: faust-update-jan-18-2023
+authors:
+  - name: Chris Wiegman
+    title: Engineering Manager
+    url: https://github.com/ChrisWiegman
+    image_url: https://avatars.githubusercontent.com/u/394675?v=4
```
Comment (TeresaGobble) [inline internal/faustjs.org/blog/2023-01-18-faust-update-jan-18-2023.md:9]:
```suggestion
    author_image_url: https://avatars.githubusercontent.com/u/394675?v=4
```

**Review by TeresaGobble — CHANGES_REQUESTED** [review]
This is awesome Chris, thank you so much! I noticed the images weren't rendering as expected: 
image

It looks like  https://docusaurus.io/docs/next/blog guide has them set with `author_image_url`. That should fix it. Thank you for this! 

image

File: internal/faustjs.org/blog/2021-12-03-upgrading-to-faustwp.md [hunk 4]
```diff
@@ -2,6 +2,11 @@
 title: Upgrading to FaustWP
 description: All you need to know about upgrading to FaustWP.
 slug: upgrading-to-faustwp
+authors:
+  - name: Andrew Matthews
+    title: Senior Software Engineer
+    url: https://github.com/apmatthews
+    image_url: https://avatars.githubusercontent.com/u/4661832?v=4
```
[the remark this answers is not in the document]
Comment (ChrisWiegman) [reply internal/faustjs.org/blog/2021-12-03-upgrading-to-faustwp.md:9]: Investing this is breaking it :(

[the remark this answers is not in the document]
Comment (ChrisWiegman) [reply internal/faustjs.org/blog/2021-12-03-upgrading-to-faustwp.md:9]: author_image_url seems to remove it entirely. It's no longer trying to render the image.

Screenshot 2023-02-01 at 15 01 41

[the remark this answers is not in the document]
Comment (TeresaGobble) [reply internal/faustjs.org/blog/2021-12-03-upgrading-to-faustwp.md:9]: Oh how odd! I wonder. I just checked it in a single blog post after pulling your branch down and it gave me the rendered image correctly- I'll check the docs out 👍

**Review by TeresaGobble — APPROVED** [review]

**Comment by TeresaGobble** [comment]
Approving changes after discussing caching- it should work fine at `image_url` 👍

**Pull request merged**